content/go/concepts/Variables/Variables.md

```diff
@@ -0,0 +1,124 @@
+
+---
+Title: 'Variables'
+Description: 'Variables are used to store numbers, strings, true/false boolean states, or objects based on their type when declaring them.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Variables'
+  - 'Booleans'
+  - 'Strings'
+  - 'Integers'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/back-end-engineer-career-path'
+---
+
+## Definition
+
+Variables are used to store data such as numbers, strings, or Booleans in memory for later use.
+
+## Difference From Constants 
+
+Unlike constants, when declaring a variable, its value can be reassigned later on but with the restriction that the new value's data type is the same as the data type when initialized.
+
+### Pseudocode Example
+
+This is acceptable
+
+```pseudo
+declare an int variable
+use it in a while loop
+reassign it to zero after
+```
+
+This will result in an error
+
+```pseudo
+declare an int variable
+use it in a while loop
+reassign it to "Codecademy" after
+```
+
+## Syntax
+
+```go
+var 'variable name' 'var type'
+```
+
+Begin with the `var` keyword followed by the name of the variable and then the variable type.
+
+Variable names should:
+
+- Begin with a letter. 
+- Must not contain special characters, only letters and digits.
+- Must not be a Go keyword. 
+- Names are case sensitive.
+ 
+Some of the basic types are:
+
+- `int`
+- `bool`
+- `string`
+
```

Comment:
```suggestion
```

If we link to the existing entry on Go data types, this part wouldn't be necessary.